src/Makefile

```diff
@@ -26,7 +26,7 @@ PROGS = ar65     \
 .SUFFIXES:
 
 bindir  := $(PREFIX)/bin
-datadir := $(if $(PREFIX),$(PREFIX)/share/cc65,$(abspath ..))
+datadir := $(PREFIX)/share/cc65
```

Comment:
This patch breaks the ability to run cc65 without installing it. That is, just compile it in-place and let it run there.

I am not sure if this feature is really needed, though.
However, one side-effect of this is that "make test" does not work as long as cc65 is not installed yet, because it does not find its files. That's why the $(abspath ..) was there.

Reply:
I regularily run cc65 from its build directory. In fact, I think I haven't "installed" it for 15 years (at least). Please preserve that possibility!

Reply:
no worries, this PR was made to see "whether something breaks, and what specifically"

Reply:
> I regularily run cc65 from its build directory. In fact, I think I haven't "installed" it for 15 years (at least). Please preserve that possibility!

Now that you say it - i don't think i have *ever* installed it :=)

Reply:
By the way GNU says `make check` should work without installation.
However we called this target "test". Maybe it's time to be standard conform.